aws/aws.go

```diff
@@ -137,6 +137,24 @@ func (a *AwsAccount) CreateAwsAccount(ctx context.Context, db models.XODB) error
 	return err
 }
 
+// UpdatePrettyAWSAccount updates an AWS account for a user. It does no error
+// checking: the caller should check themselves that the AWS account exists.
+// Only the Pretty will be updated.
+func (a *AwsAccount) UpdatePrettyAWSAccount(ctx context.Context, tx *sql.Tx) error {
```

Comment:
The convention in this project is to treat acronyms as simple words, with only the first letter capitalized, so this should be `UpdatePrettyAwsAccount`.